Series/Padovan Series/C++Sol.cpp

```diff
@@ -0,0 +1,24 @@
+int pad(int n) 
+{ 
+
+    int prevprev = 1, prev = 1, curr = 1, next = 1; 
+  
+    for (int i=3; i<=n; i++) 
+    { 
+        next = prevprev + prev; 
+        prevprev = rev; 
```

Comment:
there is a bug here
"rev is not defined"

Reply:
Oh I missed that for some odd reason , thank you for pointing it out , should be ok now